servers/services/build.gradle.kts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+import org.apache.tools.ant.taskdefs.condition.Os
```

Comment:
does gradle bundle ant? kinda surprising that gradle has no api for this

Reply:
Yea - ant's included.
There is an API `org.gradle.api.Project#ant(org.gradle.api.Action<? super org.gradle.api.AntBuilder>)` et al - also a "groovy builder" within kotlin scripts for it.